### PR TITLE
common-component

### PR DESCRIPTION
**Pull request by TaimaTask**

### CODE REVIEW

File: src/commonComponent/SigninSignup.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./SigninSignup.css";
+function CommonSigninSignup() {
+  
+    return (
+      <div>
+        <div class="commoninput">
+          <label htmlFor="username">Username</label>
```
Comment (IZablah) [inline src/commonComponent/SigninSignup.jsx]: please insert HTML Forms

File: src/commonComponent/SigninSignup.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const CommonSigninSignup = () => {
+  return (
+    <form>
+      <div class="commoninput">
+        <label htmlFor="username">Username</label>
+        <input
+          id="username"
+          type="text"
+          placeholder="username@gmail.com"
+          name="username"
```
Comment (IZablah) [inline src/commonComponent/SigninSignup.jsx]: rename username to email.

File: src/commonComponent/SigninSignup.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React,{useState} from "react";
+import "./SigninSignup.css";
+import "../commonComponent/SigninSignup";
+
+let data={
+  password: '',
+  email:''
+};
```
Comment (bqaddumi) [inline src/commonComponent/SigninSignup.jsx]: remove and use state or redux if needed

File: src/commonComponent/SigninSignup.jsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        />
+      </div>
+      <div className="commoninput">
+        <label htmlFor="password">Password</label>
+        <input
+          id="password"
+          type="password"
+          placeholder="password"
+          name="password"
+          onChange={(event)=>setPassword(event.target.value)}
```
Comment (bqaddumi) [inline src/commonComponent/SigninSignup.jsx]: move this into new function

File: src/commonComponent/SigninSignup.jsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+    </form>
+
+  );
+ 
+};
+
+export default CommonSigninSignup;
+
+export const CommonDataContext = React.createContext(data);
```
Comment (bqaddumi) [inline src/commonComponent/SigninSignup.jsx]: use redux

File: src/pages/signIn/signinPageForm.jsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from "react";
+import CommonSigninSignup from "../../commonComponent/SigninSignup.jsx";
+import { CommonDataContext } from "../../commonComponent/SigninSignup.jsx";
+import "./signinPageForm.css";
+
+const SigninForm = () => {
+  const data = useContext(CommonDataContext);
+  const hh=()=>{
+    document.getElementById("hideDiv").style.visibility ="visible";
```
Comment (bqaddumi) [inline src/pages/signIn/signinPageForm.jsx]: bad practice

File: src/pages/signIn/signinPageForm.jsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+}
+hh();
```
Comment (bqaddumi) [inline src/pages/signIn/signinPageForm.jsx]: not good

**Review by bqaddumi — CHANGES_REQUESTED** [review]

**Pull request merged**